apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,32 @@
+import { AsyncPipe, NgIf } from '@angular/common';
 import { Component, OnInit } from '@angular/core';
-
+import { Observable, of } from 'rxjs';
+import { CityStore } from '../../data-access/city.store';
+import { FakeHttpService } from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <ng-container *ngIf="cities$ | async as cities">
+      <app-card [store]="store" [getName]="getCityName" [list]="cities">
+        <img width="200px" src="/assets/img/city.avif" />
+      </app-card>
+    </ng-container>
+  `,
   standalone: true,
-  imports: [],
+  imports: [CardComponent, AsyncPipe, NgIf],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities$: Observable<City[]> = of([]);
```

Comment:
Why do you create an empty observable and you reassign it in the ngOnInit ? Assign it immediately to your stream. 

PS: use signals 😉